docs/redirects.csv

```diff
@@ -197,5 +197,7 @@ https://help.expensify.com/articles/new-expensify/workspaces/The-Free-Plan,https
 https://help.expensify.com/new-expensify/hubs/expenses/Connect-a-Bank-Account,https://help.expensify.com/articles/new-expensify/expenses/Connect-a-Business-Bank-Account
 https://help.expensify.com/articles/new-expensify/settings/Security,https://help.expensify.com/articles/new-expensify/settings/Encryption-and-Data-Security
 https://help.expensify.com/articles/expensify-classic/workspaces/reports/Currency,https://help.expensify.com/articles/expensify-classic/workspaces/Currency
+https://help.expensify.com/articles/expensify-classic/workspaces/reports/Report-Fields-And-Titles,https://help.expensify.com/expensify-classic/hubs/workspaces/
```

Comment:
Quick question since I don't really understand the change here. Why have we deleted those files? And will we direct them to `https://help.expensify.com/expensify-classic/hubs/workspaces/` page?

Reply:
yep!

Reply:
here's some additional context -- https://expensify.slack.com/archives/C02QSAC6BJ8/p1717446794685829

and you know what, i will actually update one of the redirects to https://help.expensify.com/articles/expensify-classic/reports/Automatically-submit-employee-reports (made that change in the file). 

and then the other redirect i set as https://help.expensify.com/expensify-classic/hubs/workspaces/ since https://github.com/Expensify/App/pull/43002 hasn't merged yet. i figured the main workspace hub page works fine enough for that redirect.